packages/gatsby-plugin-subfont/src/gatsby-node.js

```diff
@@ -1,22 +1,23 @@
 const path = require(`path`)
-const { execSync } = require(`child_process`)
-const shellescape = require(`shell-escape`)
+const subfont = require(`subfont`)
 
-exports.onPostBuild = ({ store }) => {
+exports.onPostBuild = async ({ store }, options) => {
   const root = path.join(store.getState().program.directory, `public`)
-  // TODO make this configurable
+
   const urlPaths = [`/`]
-  const baseArgs = [
-    `node_modules/.bin/subfont`,
-    `-i`,
-    `--no-recursive`,
-    `--inline-css`,
-    `--root`,
-    `file://${root}`,
-  ]
-  const args = baseArgs.concat(
-    urlPaths.map(currentPath => path.join(root, currentPath, `index.html`))
+
+  await subfont(
+    {
+      root: `file://${root}`,
+      inPlace: true,
+      inlineCss: true,
+      silent: true,
+      inputFiles: urlPaths.map(currentPath =>
+        path.join(root, currentPath, `index.html`)
+      ),
+      ...options,
+    },
+    console
```

Comment:
Is this intentional?
Should it be a reporter? Empty?

Reply:
could you explain some more details?

Reply:
You're passing on `console`, which I assume is to print out logging and what not. We tend to use a reporter for this purpose, as it manages the CLI output a bit. I'm not sure it should be printing anything at all but I'm not sure how it's going to be used.

Reply:
FYI: this is subfont's example output.

```
info public/404.html: 2 fonts (2 variants) in use, 8.62 MB total. Created subsets:
info   Black Han Sans:
info     400 : 146/ 2733 codepoints used, 979 kB (ttf) => 15.3 kB (woff2)
info   Jeju Myeongjo:
info     400 : 574/17718 codepoints used, 7.64 MB (otf) => 64.8 kB (woff2)
info public/index.html: 2 fonts (2 variants) in use, 8.62 MB total. Created subsets:
info Total savings: 77.6 MB
info Output written to
```

In my opinion, it might be verbose but informative so outputs should be shown to users. Anyway, it does IO jobs a lot and easy to make errors that should be notified.

I replaced console with reporter. could you review my revision?